common/signal_processing/CMakeLists.txt

```diff
@@ -4,19 +4,24 @@ project(signal_processing)
 find_package(autoware_cmake REQUIRED)
 autoware_package()
 
-ament_auto_add_library(lowpass_filter_1d SHARED
+ament_auto_add_library(lowpass_filters SHARED
   src/lowpass_filter_1d.cpp
   src/lowpass_filter.cpp
-)
+  src/butterworth.cpp)
 
 if(BUILD_TESTING)
   ament_add_ros_isolated_gtest(test_signal_processing
     test/src/lowpass_filter_1d_test.cpp
     test/src/lowpass_filter_test.cpp
-  )
+    test/src/butterworth_filter_test.cpp)
+
+  target_include_directories(test_signal_processing PUBLIC test/include)
   target_link_libraries(test_signal_processing
-    lowpass_filter_1d
-  )
+    lowpass_filters)
+
 endif()
 
 ament_auto_package()
+
+#add_executable(main_butterworth usage_examples/main_butterworth.cpp)
+#target_link_libraries(main_butterworth lowpass_filters)
```

Comment:
please remove unused comments. 
And need a new line at the end of the file. Please check other files as well.

Reply:
I will clean it out, but what about these kinds of comments? 

`double cutoff_frequency_rad_sec{};  // filter cut-off frequency [rad/sec]` 

or 

`/**
 * @brief This method assumes the continuous time transfer function of filter has already been
 * computed and stored in the object and uses the bilinear transformation to obtain the discrete
 * time transfer function.
 *
 * Matlab equivalent :
 * Td = 2.
 * [numd, dend] = bilinear(sys_filt.Numerator{1}, sys_filt.Denominator{1}, 1/Td)
 * where sys_filt is the continuous time transfer function.
 * */
void ButterworthFilter::computeDiscreteTimeTF(const bool & use_sampling_frequency)
{
  discrete_time_zeros_.resize(order_, {-1.0, 0.0});  // Butter puts zeros at -1.0 for causality
  discrete_time_roots_.resize(order_, {0.0, 0.0});`

Reply:
I have cleaned.